v23.1/data-domiciling.md

```diff
@@ -29,7 +29,7 @@ This page assumes you are already familiar with:
 
 ## Example
 
-In the following example, you will go through the process of configuring the [MovR](movr.html) data set using [multi-region SQL statements](multiregion-overview.html). Then, as part of implementing a data domiciling strategy, you will apply restricted replica settings using the [`ALTER DATABASE ... PLACEMENT RESTRICTED`](alter-database.html#placement) statement. Finally, you will verify that the resulting replica placements are as expected using [replication reports](query-replication-reports.html).
+In the following example, you will go through the process of configuring the [MovR](movr.html) data set using [multi-region SQL statements](multiregion-overview.html). Then, as part of implementing a data domiciling strategy, you will apply restricted replica settings using the [`ALTER DATABASE ... PLACEMENT RESTRICTED`](alter-database.html#placement) statement. Finally, you will verify that the resulting replica placements are as expected using the [critical nodes status endpoint](monitoring-and-alerting.html#critical-nodes-endpoint).
```

Comment:
NB @zachlite and @williamkulju Please disregard changes to the data-domiciling.md file.  I'm going to break this work out into a separate PR.  The reason is that the 'Data Domiciling' page had Legal review when it was added, so I'm going to put any subsequent changes to this page through that process.  

(The changes were added here due to oversight on my part, so I'm gonna remove them from this PR soon, so please disregard)

Reply:
Just removed this in the latest push.  Will open a new issue for the data domiciling page updates.